auth/src/VSCodeAzureSubscriptionProvider.ts

```diff
@@ -58,7 +58,8 @@ export class VSCodeAzureSubscriptionProvider extends vscode.Disposable implement
      *
      * @returns A list of tenants.
      */
-    public async getTenants(session?: vscode.AuthenticationSession): Promise<TenantIdDescription[]> {
+    public async getTenants(account?: vscode.AuthenticationSessionAccountInformation): Promise<TenantIdDescription[]> {
+        const session = await vscode.authentication.getSession(getConfiguredAuthProviderId(), ['https://management.azure.com/.default'], { account: account });

```

Comment:
We can't hardcode the management scope here since it needs to change based on the sovereign cloud setting.

Reply:
Otherwise the changes are 💯